### PR TITLE
[#49] Coming Soon page

### PR DESCRIPTION
**Pull request by krissy**
## What this does
- [x] Add  separate Coming Soon layout + temporary index route to the router keeping everything in build in a /wip subdirectory
- [x] Create content / copy as per #48, including email us button
- [x] Style to UX in #48
- [x] Create reusable button component

### Desktop:

dml_site_large

### Tablet:

dml_site_medium

### Phone:

dml_site_small

### CODE REVIEW

File: webpack/components/shared/DMLButton.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  static get propTypes() {
+    return {
+      buttonText: React.PropTypes.string,
+      buttonURL: React.PropTypes.string
+    };
+  }
+
+  render() {
+    return (
+      <div>
```
Comment (tanyapowell) [inline webpack/components/shared/DMLButton.jsx]: Could we use a `button` rather than a `div`?

**Comment by tanyapowell** [comment]
Currently blocked by #60 and the lack of responsive designs ( #48 ). Once #60 and #61 have been merged will revisit this to tidy up the `sass` architecture

##### Desktop version
image

**Comment by krissy** [comment]
@tanyapowell changes look good and fine on mobile to me too. Not sure if you saw Rocio's update to the UX of this page in https://github.com/empowerhack/DrawMyLife-Website/issues/48#issuecomment-297441140. Basically, the background image has changed and a couple of styles + link to EmpowerHack with logo. Feel free to merge this and we can do the updates  in a separate ticket, or as part of this ticket. Either way!

**Comment by tanyapowell** [comment]
Cool @krissy. If I have time tonight I'll finish it off with the new designs, if I don't manage to do it by tomorrow night (May 13th) then someone else please feel free to pick it up.

Also, #60 is no longer blocking this. So I've removed `blocked` status as there are no longer any impediments

**Comment by krissy** [comment]
@rocioleon @tanyapowell Updated with UX _close to_ the changes in https://github.com/empowerhack/DrawMyLife-Website/issues/48#issuecomment-297441140. To keep code simple and get this out quick with my rusty CSS skills, not as much variation for small & medium viewports as the mockups have but it works responsively fine. Also had to guess with some colours (and image opacity etc) using styleguide. 

@rocioleon One thing I need is an asset for the logo with white text, as could only find the dark text version in the Drive.

I'm going to merge this now in any case and get it up on GitHub pages, any other changes (and quick update of logo) can be done in a separate PR.

**Pull request merged**